cypress/e2e/tests/structureSurvey/noTrialSpecific/onboardPractice.cy.ts

```diff
@@ -0,0 +1,144 @@
+import moment from "moment";
+import "moment/min/locales";
+
+import {
+  clearCache,
+  loginWebAdmin,
+  logoutWebAdmin,
+  assignSurveyToPatient,
+  unassignSurveyToPatient,
+  calculatedTime,
+} from "../../../helpers/adminHelper/adminHelper";
+
+import {
+  verifySurveyCard,
+  ONBOARDING_PRACTICE_SURVEY_ID,
+  verifyOnboardingExperienceOptions,
+  verifyPleaseProvideFeedbackScreen,
+  verifyQuestionsNotBeenAnswered,
+  checkAlertMessage,
+  currentDate,
+  SCHEDULE_ONCE,
+  submitSurvey,
+  CONTAINED_PRIMARY_BUTTON,
+  ALERT_NOTIFICATION,
+  verifyInputMessageError,
+  OPTION_RADIO_EXCELLENT,
+  FEEDBACK_TEXT_INPUT,
+  QUESTIONS_TEXT_INPUT,
+  verifyOnboardPracticeConfirmation,
+} from "../../../helpers/surveyHelper";
+
+import {
+  loginWebPatient,
+  logoutWebPatient,
+} from "../../../helpers/loginHelper";
+
+import {
+  ONBOARDING_PRACTICE_SURVEY,
+  THIS_FIELD_CANNOT_BE_LEFT_BLANK,
+} from "../../../strings/surveyStrings";
+
+import { SURVEY_CARD_BUTTON } from "../../../helpers/homeHelper";
+import { NEXT } from "../../../helpers/generalStrings";
+
+describe("notTrialSpecific - Onboarding Practice Survey", () => {
+  beforeEach(() => {
+    clearCache();
+    loginWebAdmin(Cypress.env("emailAdmin"), Cypress.env("passwordAdmin"));
+    assignSurveyToPatient("Alan Patient 20", ONBOARDING_PRACTICE_SURVEY);
+    logoutWebAdmin();
+
+    clearCache();
+    loginWebPatient(
+      Cypress.env("emailWebPatient"),
+      Cypress.env("passwordWebPatient"),
+    );
+
+    cy.get(SURVEY_CARD_BUTTON("Surveys"), { timeout: 10000 })
+      .should("be.visible")
+      .click();
+  });
+
+  afterEach(() => {
+    logoutWebPatient();
+    clearCache();
+    loginWebAdmin(Cypress.env("emailAdmin"), Cypress.env("passwordAdmin"));
+    unassignSurveyToPatient("Alan Patient 20", ONBOARDING_PRACTICE_SURVEY);
+    logoutWebAdmin();
+  });
+
+  it("How has your onboarding experience been so far? should be required, #5783", () => {
+    cy.get(ONBOARDING_PRACTICE_SURVEY_ID).click();
+    cy.get(CONTAINED_PRIMARY_BUTTON).contains(NEXT).click();
+    cy.wait(2000);
+    checkAlertMessage("Please select one option");
```

Comment:
We could add it to strings
And before calling check function we could add a step to verify we are on the correct screen (Same for the feedback screen)

Reply:
Ok, I'm not sure I understand, but I inserted a step to check if you are on the correct page before and after error validation.

Reply:
Right! I would suggest to add the validation of screen before triggering the error